pythonsdk/build.gradle

```diff
@@ -13,13 +13,22 @@ ext {
         pypi_repo = 'testpypi'
     }
     if (!project.rootProject.hasProperty('vantiqServiceSdkVersion')) {
-        vantiqServiceSdkVersion = '0.0.1'
+        vantiqServiceSdkVersion = '0.0.2'
     }
     
     // See if we are using virtualenvwrapper and if so put the virtualenv in the WORKON_HOME
     if (System.getenv('WORKON_HOME')) {
         virtualenvPath = System.getenv('WORKON_HOME') + "/vantiqservicesdk"
     }
+
+    String depList = ''
```

Comment:
Avoids duplication of the SDK requirements in both `pyproject.toml` and `requirements.txt`.  Both are generated from the SDK requirements.